cmd/clusterawsadm/cloudformation/bootstrap/cluster_api_controller.go

```diff
@@ -387,6 +387,10 @@ func (t Template) ControllersPolicy() *iamv1.PolicyDocument {
 		})
 	}
 
+	if t.Spec.ClusterAPIControllers.ExtraStatements != nil {
```

Comment:
We should not change the `controllers.cluster-api-provider-aws.sigs.k8s.io` here, instead similar to controlplane and node, we want to attach extra statemements as `inline policy`: https://github.com/kubernetes-sigs/cluster-api-provider-aws/blob/2a183f580873ed46aebdbba77da212a3476682aa/cmd/clusterawsadm/cloudformation/bootstrap/control_plane.go#L24

And add this as inline policy in the `Policies` section of the controller role.

Reply:
Oh yea, that totally makes sense! Sorry about that, fixed.